ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenSpec.java

```diff
@@ -125,6 +124,9 @@ void addFunction(final FunctionName functionName, final Kudf function) {
 
     void addStructSchema(final CreateStructExpression struct, final Schema schema) {
       final String structSchemaName = CodeGenUtil.schemaName(structSchemaCount++);
+      if (structToSchemaName.containsKey(struct)) {
```

Comment:
let's move the if branch one line up (to make sure that we don't unnecessarily increment `structSchemaCount`)